contents/blog/the-posthog-array-1-32-0.md

```diff
@@ -0,0 +1,133 @@
+---
+date: 2022-01-31
+title: Array 1.32.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes"]
+featuredImage: ../images/blog/posthog-array-blog.png
+excerpt: Excerpt coming soon.
+---
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Do not upgrade to this version if you have deployed PostHog using Postgres. PostHog no longer supports Postgres as of <code>v1.31.0</code> (last version supported is <code>1.30.0</code>) and you must <a href="/docs/self-host/migrate-from-postgres-to-clickhouse" target="_blank">upgrade to ClickHouse</a> first.
```

Comment:
PostHog definitely supports Postgres (kinda doesn't work without it). More accurately, it no longer works without ClickHouse. Or, differently worded, Postgres-based installations must be upgraded to become ClickHouse-based.